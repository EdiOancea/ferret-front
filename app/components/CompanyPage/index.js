import React from 'react';
import PropTypes from 'prop-types';
import ViewHeadlineOutlinedIcon from '@material-ui/icons/ViewHeadlineOutlined';

import PaperFrame from 'components/PaperFrame';
import ViewCompanyContainer from 'containers/ViewCompany';

const CompanyPage = props => (
  <PaperFrame
    headerText="Company Information"
    icon={<ViewHeadlineOutlinedIcon />}
  >
    <ViewCompanyContainer id={props.match.params.id} />
  </PaperFrame>
);

CompanyPage.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      id: PropTypes.string.isRequired,
    }),
  }),
};

export default CompanyPage;
