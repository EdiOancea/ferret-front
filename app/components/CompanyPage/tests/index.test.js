import React from 'react';
import { shallow } from 'enzyme';

import CompanyPage from 'components/CompanyPage';

describe('<CompanyPage />', () => {
  it('should not regress', () => {
    const config = {
      params: {
        id: '1',
      },
    };
    const tree = shallow(
      <CompanyPage match={config}>
        <div>a</div>
      </CompanyPage>,
    );
    expect(tree.children.length).toBe(1);
    expect(tree).toMatchSnapshot();
  });
});
