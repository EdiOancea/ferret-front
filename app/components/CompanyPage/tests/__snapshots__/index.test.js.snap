// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CompanyPage /> should not regress 1`] = `
<WithStyles(PaperFrame)
  headerText="Company Information"
  icon={<pure(ViewHeadlineOutlinedIcon) />}
>
  <withSaga(withReducer(Connect(ViewCompany)))
    id="1"
  />
</WithStyles(PaperFrame)>
`;
